infra/modules/service/draft_documents.tf

```
@@ -9,7 +10,8 @@ resource "aws_s3_bucket" "draft_documents" {
 }
 
 resource "aws_s3_bucket_public_access_block" "draft_documents" {
-  bucket = aws_s3_bucket.draft_documents.id
+  count  = startswith(var.service_name, "api-") ? 1 : 0
+  bucket = one(aws_s3_bucket.draft_documents[*].id)
```

While this does work, the usual pattern here is

```suggestion
  bucket = aws_s3_bucket.draft_documents[0].id
```

and consistency is nice!

---

I had seen some folks saying those [0].blah would fail if there was nothing in the set? But I guess if it works elsewhere and we have stuff turned off then maybe that is handled better now.